mage_ai/streaming/sinks/elasticsearch.py

```diff
@@ -52,8 +53,17 @@ def write(self, message: Dict):
     def batch_write(self, messages: List[Dict]):
         self._print(f'Batch ingest data {messages}, time={time.time()}')
         if self.config._id is not None:
-            docs = [{'_index': self.config.index_name, '_id': doc[self.config._id],
-                     'doc': doc} for doc in messages]
+            if self.config._op_type is not None:
```

Comment:
I would simplify the code like these
```
docs = []
for msg in in messages:
    doc = {'_index': self.config.index_name, 'doc': doc}
    if self.config._id is not None:
        doc['id'] = msg[self.config._id]
    if self.config._op_type is not None:
        doc['_op_type'] = self.config._op_type
    docs.append(doc)
```

Reply:
Your right this looks better and simpler :-)